Add documentation how to run a node from snapshot (#3183)
aeternity/aeternity: README.md

    aeternity/aeternity
```

#### Restore from snapshot

> To speedup the initial blockchain syncronization the node database can be restored from a snapshot following the below steps:

Reviewer: synchronization

Author: 92ff9b7

* delete the contents of the database if the node has been started already
* download the database snapshot
* verify if the snapshot checksum match the downloaded file
* unarchive the database snapshot

**Note that the docker container must be stopped before replacing the database**

> The following snipped can be used to delete current database and replace it the latest mainnet database snapshot and restore it assuming the database path is ` ~/.aeternity/maindb`:

Reviewer: ```suggestion
The following snippet can be used to replace the current database with the latest mainnet snapshot assuming the database path is ` ~/.aeternity/maindb`:
```

Author: 92ff9b7
